client/components/docs/docs.service.js

```diff
@@ -1,48 +1,51 @@
-class DocsService {
-    constructor ($translate, TranslateService, TARGET, DOCS_ALL_GUIDES, DOCS_HOMEPAGE_GUIDES) {
-        this.$translate = $translate;
-        this.TranslateService = TranslateService;
-        this.TARGET = TARGET;
-        this.DOCS_ALL_GUIDES = DOCS_ALL_GUIDES;
-        this.DOCS_HOMEPAGE_GUIDES = DOCS_HOMEPAGE_GUIDES;
-    }
+{
+    class DocsService {
+        constructor ($translate, OvhApiMe, TranslateService, TARGET, DOCS_ALL_GUIDES, DOCS_HOMEPAGE_GUIDES) {
+            this.$translate = $translate;
+
+            this.OvhApiMe = OvhApiMe;
+            this.TranslateService = TranslateService;
 
-    getDomainOfGuides () {
-        if (this.TARGET === "US") {
-            return "US";
+            this.TARGET = TARGET;
+            this.DOCS_ALL_GUIDES = DOCS_ALL_GUIDES;
+            this.DOCS_HOMEPAGE_GUIDES = DOCS_HOMEPAGE_GUIDES;
         }
-        const locale = this.TranslateService.getGeneralLanguage();
 
-        if (locale === "fr") {
-            return "FR";
+        fetchSubsidiary () {
+            return this.OvhApiMe.v6()
+                .get().$promise
+                .then(me => me.ovhSubsidiary);
         }
-        return "EN";
-    }
 
-    getAllGuidesLink () {
```

Comment:
All you had to do is to change this function to handle ovhSubsidiary.

Reply:
Actually no, I'll have to make it work with #805